src/themes/default.css

```diff
@@ -4,14 +4,30 @@
   /* Colors */
   --gr-color-primary: #f27a1a;
   --gr-color-primary-hover: #ef6114;
+  --gr-color-primary-outline: #FFF;
   --gr-color-secondary: #273142;
   --gr-color-secondary-hover: #0f131a;
-  --gr-color-success: #0bc15c;
-  --gr-color-success-hover: #09a44e;
-  --gr-color-content-primary: #273142;
-  --gr-color-content-secondary: #6e7787;
-  --gr-color-content-tertiary: #95a1b5;
-  --gr-color-content-passive: #afbbca;
+  --gr-color-secondary-outline: #FFF;
```

Comment:
Outline buttons has always `transparent` backgrounds. So I think we should not need those variables.

Reply:
Known issue. It will be solved in the next commit 👍